config/webpack/loaders/erb.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  test: /\.erb$/,
```

Comment:
Webpacker default :).

Realistically, I think we have zero use for `erb` here, but we probably will need to do a `haml` pack - right now, every angular directive/component template (anything referenced by `templateUrl`) lives in `app/views/static/` and we have a special controller that just serves those, passing the template through gettext and haml->html conversions ... but in the future, we should probably compile those on the server and do per-language template bundles..